### PR TITLE
automatic cert renewal for observability

### PR DESCRIPTION
**Pull request by dockerymick**
[Doc issue](https://github.com/open-cluster-management/backlog/issues/7125)

[Dev issue](https://github.com/open-cluster-management/backlog/issues/4766)

### CODE REVIEW

File: security/certificates.adoc [hunk 1]
```diff
@@ -44,7 +44,7 @@ oc delete secret grc-0c925-grc-secrets -n open-cluster-management
 [#refresh-certificates-ca]
 == Refresh certificates for {product-title}
 
-You can refresh all certificates that are issued by the {product-title-short} CA. During the refresh, the Kubernetes secret that is associated with each `cert-manager` certificate is deleted. The service restarts automatically to use the certifcate. Run the following command:
+You can refresh all certificates that are issued by the {product-title-short} CA. During the refresh, the Kubernetes secret that is associated with each `cert-manager` certificate is deleted. The service restarts automatically to use the certificate. Run the following command:
```
Comment (marcolan018) [inline security/certificates.adoc:47]: user don't need to run any command for cert renew, or restart any service to apply the renewed cert

Comment (dockerymick) [reply security/certificates.adoc:47]: This file is for RHACM certificates. I was updating the spelling on line 47. @marcolan018 may you review the blurb that i added in the observability/observe_intro.adoc  file?

File: observability/observe_intro.adoc [hunk 2]
```diff
@@ -20,7 +20,20 @@ When the service is enabled the `observability-endpoint-controller` is automatic
...
 The observability service deploys an instance of Prometheus AlertManager, which enables alerts to be forwarded with third-party applications. It also includes an instance of Grafana to enable data visualization via dashboards (static) or data exploration. 
 
-Customize the observability. You can create custom link:https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/[recording rules] or link:https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[alerting rules]. 
+Observability certificates are automatically renewed upon expiration. View the following list to understand the effects when certificates are automatically renewed:
+
+* Components on your hub cluster automatically restart to retrieve the renewed certificate.
+
+* Manifest work for observability is updated.
+
+// is manifest work the multicluster-observability-operator yaml, the configured object store?
```
Comment (dockerymick) [inline observability/observe_intro.adoc]: @marcolan018 can you clarify what the manifest work? Are there specific YAML files that we need to mention when we let the user know what work is being updated?

Comment (marcolan018) [reply observability/observe_intro.adoc]: manifestwork is about the internal implementation about how the artifacts in hub side pushed to managed clusters. Can we update this step to something like "The renewed certificates are pushed to managed clusters by ACM"?

**Comment by dockerymick** [comment]
@marcolan018 I have updated the file, may you rereview the PR?

**Review by marcolan018 — APPROVED** [review]
lgtm

**Pull request merged**